src/main/java/com/saralein/server/connection/ConnectionSocket.java

```diff
@@ -21,9 +21,20 @@ public String read() throws IOException {
         BufferedReader bufferedReader = new BufferedReader(in);
 
         String str;
+        int length = 0;
 
         while ((str = bufferedReader.readLine()) != null && !str.isEmpty()) {
             request.append(str).append(CRLF);
+
+            if (str.startsWith("Content-Length")) {
+                length = Integer.parseInt(str.split(":")[1].trim());
+            }
+        }
+
+        if (length > 0) {
+            char[] body = new char[length];
+            bufferedReader.read(body);
+            request.append(CRLF + "Body: " + new String(body));
```

Comment:
This is the spot where I'm trying to decide if this should move to the parse, and if the parser should receive the `BufferedReader`.